mmv1/templates/terraform/examples/scc_event_threat_detection_custom_module.tf.erb

```diff
@@ -0,0 +1,18 @@
+resource "google_scc_event_threat_detection_custom_module" "<%= ctx[:primary_resource_id] %>" {
+  organization = "<%= ctx[:test_env_vars]['org_id'] %>"
+  display_name = "<%= ctx[:vars]['display_name'] %>"
+  enablement_state = "ENABLED"
+  type = "<%= ctx[:vars]['type'] %>"
+  description = "My Event Threat Detection Custom Module"
+  cofig = jsonencode({
+    "metadata": {
+      "severity": "LOW",
+      "description": "Flagged by Forcepoint as malicious",
```

Comment:
Just want to make sure this makes sense being shown as the example at the top of the documentation page. Same with the IPs below.

Reply:
It is an example in https://cloud.google.com/security-command-center/docs/use-custom-modules-etd#template_structure

Reply:
Cool, makes sense, thanks for verifying!